contrib-chart/README.md

```diff
@@ -0,0 +1,37 @@
+# Bring Your Own Chart
+
+We at Devtron strongly believe that a product is always incomplete without the contribution of it's community and the users. What else can be better than devtron users becoming a part of devtron team, the product they love the most and here we are giving you an opportunity to contribute to an Open Source product by sharing your own helm charts with us.
```

Comment:
product they love.

we cant use sentence like.. we are giving you an opportunity to contribute...  rather user something like

Through this upcoming feature you can help other community members of Devtron by contributing your helm charts to Devtron.

Reply:
Changes done as suggested